libs/blocks/hero-marquee/hero-marquee.css

```diff
@@ -260,6 +260,41 @@ html[dir="rtl"] .hero-marquee li.icon-item span.icon {
   order: 2;
 }
 
+/* icons */
+.hero-marquee .icon-list.has-svg-bullet {
+  padding-inline-start: 0;
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacing-xs);
+}
+
+.hero-marquee .icon-list.has-svg-bullet li.icon-item {
+  display: flex;
+  gap: 12px;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+.hero-marquee .row-list.align-icon-start li.icon-item {
+  align-items: flex-start;
+}
+
+.hero-marquee li.icon-item picture.list-icon,
+.hero-marquee li.icon-item picture.list-icon img {
+  display: flex;
+  max-width: 36px;
+  height: auto;
+}
+
+.hero-marquee .body-bold {
+  font-weight: 700;
+}
```

Comment:
I don't remember why this was needed 🤔 It's not currently defined in the codebase, I don't see it used on the test page, nor can I see a JS reference to it. Is this still relevant?